andela/ah-92explorers-backend
# #166789990 User Should Be Able To Receive Verification Email After Signup

#### What does this PR do?

- Verify user account

#### Description of Task to be completed

- User should be able to receive a verification link via email upon successful signup.

#### How should this be manually tested?

1. Run the following commands in Terminal

```
git clone git@github.com:andela/ah-92explorers-backend.git
cd ah-92explorers-backend
git checkout ft-verification-email-166789990
npm install
npm start
```

2. Add environment variables as specified in `.env.sample` file

3. Go to the postman and Register a user via `http://localhost:3000/api/v1/signup`, 

4. A verification email will be sent to the user after successfully signing up.

5. Go to your email and verify user account via a button `Verify Account`.

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[166789990](https://www.pivotaltracker.com/story/show/166789990)
#### Screenshots (if appropriate)

- After successful registration, the user should receive a verification email.
<img width="768" alt="Screen Shot 2019-07-07 at 19 12 33" src="https://user-images.githubusercontent.com/40357462/60771628-3cd71c80-a0eb-11e9-8c62-866339c5eedf.png">

- Through the email, user should click on `Verify Account` button.
<img width="830" alt="Screen Shot 2019-07-06 at 21 52 13" src="https://user-images.githubusercontent.com/40357462/60762295-c12e8e80-a05c-11e9-88ad-54a387397d75.png">

- Finally, Account will be verified.
<img width="1320" alt="Screen Shot 2019-07-06 at 21 52 46" src="https://user-images.githubusercontent.com/40357462/60762296-c1c72500-a05c-11e9-9358-9791a52d8d91.png">

#### Checklist
